INSTRUCTION.md

```diff
@@ -0,0 +1,30 @@
+# Instructions on how to deploy.
+
+## Deployment of DaemonSet:
+```
+kubectl apply -f ./.infrastructure/daemonset.yml
+```
+#### Verify deployment: DaemonSet
+```
+kubectl get pods -n mateapp
+```
+#### To view the logs of DaemonSet pods, use the command:
+```
+kubectl logs <pod-name> -n mateapp
```

Comment:
The namespace `mateapp` is used here, but the DaemonSet configuration specifies the `todoapp` namespace. Ensure the namespace matches the configuration file.